### PR TITLE
Addition of multiple Baby Driver quotes in English

### PR DESCRIPTION
**Pull request by Zarrten**
I have added multiple quotes from the 2017 movie Baby Driver under English.

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38807,6 +38807,30 @@
       "source": "A Rushing River Speaks - Quiet Commotion",
       "id": 6758,
       "length": 119
+    },
+    {
+      "text": "He's been boosting cars since he was old enough to see over the dash. He stole my Mercedes. Had a lot of merch in it. I watched him do it, too. I didn't stop him though, because I was just blinded by the balls on that kid.",
+      "source": "Baby Driver",
+      "id": 6760,
+      "length": 222,
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38815]: Remove

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38807,6 +38807,30 @@
...
+    {
+      "text": "Well, then, you have us all beat. Every damn song is about you. We could drive back and forth across the States forever and never run out of Baby songs.",
+      "source": "Baby Driver",
+      "id": 6762,
+      "length": 152,
+    }
+    {
+      "text": "You rob to support a drug habit, I do drugs to support a robbery habit.",
+      "source": "Baby Driver",
+      "id": 6763,
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38832]: Remove

**Review by Miodec — CHANGES_REQUESTED** [review]
Missing commas between quotes

File: frontend/static/quotes/english.json (hunk 1 above)
Comment (Zarrten) [reply frontend/static/quotes/english.json]: I have removed the mentioned quotes

**Pull request merged**